gtda/images/filtrations.py

```diff
@@ -349,3 +350,135 @@ def transform(self, X, y=None):
         Xt = np.concatenate(Xt)
 
         return Xt
+
+
+@adapt_fit_transform_docs
+class DilationFiltration(BaseEstimator, TransformerMixin):
+    """Filtrations of 2D/3D binary images based on distances to activated
+    pixels.
+
+    The dilation filtration assigns to each deactivated pixel of an image a
+    pixel value corresponding to the Manhattan distance between the pixel and
+    its closest activated pixel. The dilation process is iterative and the
+    number of iterations `n_iterations` sets the maximum distance considered.
```

Comment:
The first and second sentences seem a little at odds to me. The first sentence does not suggest that there should be any iterations at all, so what exactly is the role of the iteration? Are we thickening the activated regions "by 1" for each iteration, and remembering how many thickenings are needed to get to each non-activated pixel?

Reply:
We dilate iteratively, first to the neighboring pixels, then to the 2nd layers of neighbors etc... `n_iterations` times. We are thickening the activated region by 1 at each iteration indeed. Let me propose this modification

> """Filtrations of 2D/3D binary images based on the thickening of activated regions.

> The dilation filtration assigns to each deactivated pixel around an activated pixel of an image a pixel value corresponding to the Manhattan distance between the pixel and
its closest activated pixel. The number of iterations `n_iterations` defines the thickness of the dilation. Undilated deactivated pixels are assigned the maximum Manhattan distance between two pixels on the image and activated
    pixels are assigned a value of 0.